src/chrome/content/rules/Necessary_and_Proportionate.org.xml

```diff
@@ -2,7 +2,7 @@
 	For other EFF coverage, see EFF.xml.
 
 -->
-<ruleset name="Necessary and Proportionate.org">
```

Comment:
This domain works. Maybe your curl script ran into a transient failure? Would be good to run again and check the difference, in case other domains got caught by transient failures.